README.md

```diff
@@ -102,3 +102,63 @@ Cache will be stored as a file in `~/.aws_fuzzy_finder.cache`.
 If you have to access your instances through a gateway instance, use the `--tunneling` param. This will make the fuzzy find to run twice: first time you will pick the gateway to tunnel through, and the second time you choose is the instance you would like to SSH into.
 
 Gateway must be allowed to access the instance with its own ssh key. You may set the user and key path spearately using `--tunnel-user` and `--tunnel-key-path` params. The key will be looked up ON the gateway instace.
+
+## Dependency conflicts/ Virtualenvs
+
+It's 2018, Summer and you are using `pip` to install `aws-fuzzy-finder` along with `awscli` on the same machine, but it fails, you see errors about botocore, you become sad.
+You want to be able to use both tools in a convenient and easy way. You accept that some initial configuration is OK, and a happy life afterwards is great, you are no longer sad.
+
+Firstly we will need, python setup tools and virtualenv as provided by your distribution. (I assume you are using a Linux).
+
+We will modify our pip config, so that packages installed with pip are always installed locally to / for the user, because you probably will not be running aws-fuzzy-finder as a system service.
+
+
+Create a file in your home >> .config >> pip directory:
+
+```
+~/.config/pip/pip.conf
+```
+
+add this to the file
+
+```
+[install]
+user = yes
+no-binary = :all:
```

Comment:
@egidijus Is this needed for this setup to work?

Otherwise, love it!

Reply:
The idea is that you will install every python package "user local" instead of system-wide.
I used to install things system-wide, but I noticed that many low-level system tools use python, and messing with dependency tree for `apt-get` can kill apt, or `yum`, or `pacman`.

So the workflow for people is to try and install things in to virtual envs, if not virtual envs, then at least in to user "local" home binaries.

As for "services", if you are using alpine and docker, you should probably be installing system-wide or in a virtual env.

Reply:
good practice :+1:

Reply:
Yeah I get the whole workflow, was only curious about `no-binary = :all:`. If I'm not mistaken this forces every package to be compiled from source instead of relying on e.g., readily available wheels. Sounds like this might be problematic in some cases (vide macOS) while not strictly required for the whole workflow/separation to work. Anyways, thanks for the change!